packages/go_router_builder/README.md

```diff
@@ -1,34 +1,5 @@
 ## Usage
 
-### Dependencies
```

Comment:
Why remove these parts?

Reply:
[Stuart said to remove the "Import" sections from the READMEs.](https://github.com/flutter/packages/pull/5478#discussion_r1424584330). I assumed that would also apply to Dependencies.

Reply:
installing https://pub.dev/packages/go_router_builder/install page seems to be generic, and does not really work for build_runner.

Reply:
I added a separate `.yaml` file to store the code snippet for the "Dependencies" section since the existing one uses a relative path for `go_router_builder`. I'm not sure if this was the best way to do it though.

Reply:
> installing https://pub.dev/packages/go_router_builder/install page seems to be generic, and does not really work for build_runner.

That's https://github.com/dart-lang/pub-dev/issues/3403, FYI.

Reply:
> I added a separate `.yaml` file to store the code snippet for the "Dependencies" section since the existing one uses a relative path for `go_router_builder`. I'm not sure if this was the best way to do it though.

This doesn't actually accomplish anything, unfortunatetly, since there's no mechanism by which a random `.yaml` file would be kept updated. Since we don't have an authoritative source to pull the excerpt from, you can just not except for that block. The tool doesn't enforce excerpts for `yaml`.